packages/trader/src/Modules/Trading/Containers/trade.jsx

```diff
@@ -66,6 +70,19 @@ class Trade extends React.Component {
         });
     };
 
+    onTrySyntheticIndicesClick = () => {
+        this.setState(
+            {
+                try_synthetic_indices: true,
+            },
+            () => {
```

Comment:
Why is `try_synthetic_indices` set to false right after it's set to true?

Reply:
@oskar-binary I need to trigger it for a second it would only open the market selector, then return to false, to avoid opening it again for any component update